### DISCUSSION TITLE
Why does a fresh installation of Panopticon 1.2.1 no longer convert .po to .ini?

### DISCUSSION THREAD

**Discussion by morphinestyle** [akeeba/panopticon#776] [Q&A (Community Support)]
I just did a clean install of Panopticon 1.2.1 and it no longer automatically converts .po to .ini.

I looked in the documentation to see if I had ever forgotten to activate some variable but I couldn't find anything, not even looking at the more "dated" installation which instead does it perfectly.

Am I forgetting something?

**Comment by nikosdion** [comment]
It never did. See https://github.com/akeeba/panopticon/wiki/Translate_Existing_Language#testing-your-translation
